app/_hub/kong-inc/saml/_index.md

```diff
@@ -288,21 +334,104 @@ params:
         - `regenerate`: generates a new session identifier on each modification and does not use expiry for signature verification. This is useful in single-page applications or SPAs.
     - name: session_compressor
+      maximum_version: "3.1.x"
       required: false
       default: none
       datatype: string
       description: |
         The session strategy:
         - `none`: no compression.
         - `zlib`: use Zlib to compress cookie data.
+    - name: session_audience
+      minimum_version: "3.2.x"
+      required: false
+      default: '"default"'
+      datatype: string
+      description: The session audience e.g. "my-application"
+    - name: session_remember
+      minimum_version: "3.2.x"
+      required: false
+      default: false
+      datatype: boolean
+      description: Enables or disables persistent sessions
+    - name: session_remember_cookie_name
+      minimum_version: "3.2.x"
+      required: false
+      default: '"remember"'
+      datatype: string
+      description: Persistent session cookie name
+    - name: session_remember_rolling_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 604800
+      datatype: integer
+      description: Persistent session rolling timeout in seconds.
+    - name: session_remember_absolute_timeout
+      minimum_version: "3.2.x"
+      required: false
+      default: 2592000
+      datatype: integer
+      description: Persistent session absolute timeout in seconds.
+    - name: session_request_headers
+      minimum_version: "3.2.x"
+      required: false
+      default: null
+      datatype: array of string elements
+      description: |
+        List of information to include (as headers) in the request to upstream. 
+        Accepted values are: `id`, `audience`, `subject`, `timeout`, `idling-timeout`, `rolling-timeout`,
+        `absolute-timeout`.
+        E.g. { "id", "timeout" } will set both `Session-Id` and `Session-Timeout` in the request headers.
+    - name: session_response_headers
+      minimum_version: "3.2.x"
+      required: false
+      default: null
+      datatype: array of string elements
+      description: |
+        List of information to include (as headers) in the response to downstream. 
+        Accepted values are: `id`, `audience`, `subject`, `timeout`, `idling-timeout`, `rolling-timeout`,
+        `absolute-timeout`.
+        E.g. { "id", "timeout" } will inject both `Session-Id` and `Session-Timeout` in the response headers.
+    - name: session_store_metadata
+      minimum_version: "3.2.x"
+      required: false
+      default: false
+      datatype: boolean
+      description: | 
+        Configures whether or not session metadata should be stored.
+        This includes information about what are the active sessions for a specific audience
+        belonging to a specific subject.
```

Comment:
Would the "specific audience" and "specific subjects" be determined by the following params?

Reply:
only the audience is configurable, the subject is inferred automatically. The configuration for the audience is controlled by the parameter: `session_audience`